packages/storage/__tests__/internals/apis/getProperties.test.ts

```diff
@@ -0,0 +1,55 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import { AmplifyClassV6 } from '@aws-amplify/core';
+
+import { getProperties as advancedGetProperties } from '../../../src/internals';
+import { getProperties as getPropertiesInternal } from '../../../src/providers/s3/apis/internal/getProperties';
+
+jest.mock('../../../src/providers/s3/apis/internal/getProperties');
+const mockedGetPropertiesInternal = jest.mocked(getPropertiesInternal);
+
+describe('advancedGetProperties', () => {
```

Comment:
Nit, but `advancedGetProperties` doesn't map to a real API name, maybe something like:

```suggestion
describe('getProperties (internal)', () => {
```